smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/model/ShapeExt.kt

```diff
@@ -135,6 +135,24 @@ fun Model.getNestedShapes(shape: Shape): Set<Shape> {
         .select(this)
 }
 
```

Comment:
These model selectors are used to extract the needed subset of shapes from the model for use above.